requirements.txt

```diff
@@ -1,53 +1,53 @@
-Django>=1.11,<2.0
-psycopg2-binary==2.7.4
+Django==3.1.3
+psycopg2-binary==2.8.6
 django-confy==1.0.4
-django-bootstrap3==10.0.1
-openpyxl==2.4.2
-django-reversion==1.10
-pytz>=2016.6.1
-django-timezone-field==2.0
-requests==2.11
+django-bootstrap3==14.2.0
+cython==0.29.21
+openpyxl==3.0.5
+django-reversion==3.0.8
+pytz==2020.4
+django-timezone-field==4.0
+requests==2.24.0
 Unipath==1.1
-six==1.10
-python-dateutil==2.6.0
-future==0.16.0
+python-dateutil==2.8.1
+future==0.18.2
 unicodecsv==0.14.1
 
 # frictionless datapackage and tableschema
-datapackage==1.2.6
-tableschema==1.0.13
+datapackage==1.15.1
+tableschema==1.20.0
 
 # rest API
-djangorestframework==3.8.2
+djangorestframework==3.12.2
 coreapi==2.3.3
 # next line is to fix a strange dep error from (the openapi-codec==1.3.1 create issues)
-django-filter==1.1.0
-django-crispy-forms==1.6.1
-django-cors-headers==1.3.1
-dry-rest-permissions==0.1.8
-djangorestframework-gis==0.13
-drf-yasg==1.7.4
-django-extra-fields==1.0.0
-Pillow==5.1.0
-djoser==1.2.2
+django-filter==2.4.0
+django-crispy-forms==1.9.2
+django-cors-headers==3.5.0
+dry-rest-permissions==0.1.10
+djangorestframework-gis==0.16
+drf-yasg==1.20.0
+django-extra-fields==3.0.2
+Pillow==8.0.1
+djoser==2.1.0
 
 # for S3 static/media storage
-django-storages==1.6.6
-boto3==1.7.50
+django-storages==1.10.1
+boto3==1.16.14
 
 # TODO: to get rid of
-django-grappelli==2.8.2
+django-grappelli==2.14.2
 
 
 # suspicious dependencies
 dj-static==0.0.6
 
 # test, deploy and dev
-coverage>=4.2,<5.0
-flake8>=2.4.0,<3.0
-mixer>=5.4.1,<6.0
-django-extensions>=1.7.2,<1.8
-ipython>=5.1.0,<6.0
-Werkzeug>=0.10.4,<1.0
-factory-boy==2.11.1
-freezegun==0.3.11
+coverage==5.3
+flake8==3.8.4
+mixer==6.1.3
+django-extensions==3.0.9
+ipython==7.19.0
```

Comment:
we should not include ipython in requirements. This is just something I include most of the time for development but I add it manually.